### DISCUSSION TITLE
TP5 - Ejercicio 2 - TaTeTi - Uso de scanf en funciones

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#156] [Preguntas y respuestas] [answered]
Ejercicio TaTeTi: Podemos usar scanf's en las funciones para que puedan ingresar los movimientos o tienen que ser en el main.

**Comment by martinvilu** [comment, marked as answer]
En el `main`, pero una pregunta, ¿como se llamaría la función que usa el `scanf`?

**Reply by Nikzza** [reply]
Perfecto. Nono, era una funcion en general, no seria una especificamente para usar scanf.

**Reply by martinvilu** [reply]
Fíjate que sin un nombre en particular, puede ser _tranquilamente_ el `main`.

**Reply by Nikzza** [reply]
sip, por eso pregunte. ya lo cambie

**Comment by martinvilu** [comment]
Duplicado de https://github.com/INGCOM-UNRN-P1/cursada-2023/discussions/153
